tests/test_cli/dash_test_script.txt

```diff
@@ -1,5 +1,5 @@
 # from manim import *
 class Dash_Test(Scene):
```

Comment:
Nitpick : you should rename this. This syntax with _Test is used by pytest to detect if a function or a class is a test. 
Maybe rename it to DashTest or to something more explicite like ExplicitCodeFlagTest (ok the name sucks but you get the gist)

Reply:
I see. In the latest commit, I've renamed it to `DashAsNameTest`.